src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -350,7 +350,7 @@ public static function trustXSendfileTypeHeader()
      *
      * @return BinaryFileResponse
      */
-    public function deleteFileAfterSend($shouldDelete)
+    public function deleteFileAfterSend($shouldDelete = true)
```

Comment:
This is a BC break forbidden by [our BC promise](http://symfony.com/doc/current/contributing/code/bc.html#changing-classes)